reference/forms/types/repeated.rst

```diff
@@ -177,6 +178,21 @@ type
 The two underlying fields will be of this field type. For example, passing
 ``PasswordType::class`` will render two password fields.
 
+compare
+~~~~~~~
+
+**compare**: ``callable`` **default**: ``null``
+
+The two underlying fields' values will be compared by this callable. The callable receives two values to compare, and should return a boolean: true if the fields are considered equal, false otherwise.
+
+When this option has a null value, it uses internally a callable comparable to:
```

Comment:
No not really. The alternative to the double colon would be the `code-block` directive like this:

```rst
.. code-block:: php

    $builder->add('password', RepeatedType::class, array(
        'compare'  => function ($value1, $value2) {
            return $value1 === $value2;
        },
    ));
```

But throughout the docs we prefer the double colon if the previous paragraph ended with a colon and we want to show a PHP code example.